### DISCUSSION TITLE
frozenset to HashSet<T>

### DISCUSSION THREAD

**Discussion by hauntsaninja** [PyO3/pyo3#3771] [Questions]
PyO3's conversion guide https://pyo3.rs/v0.11.0/conversions#argument-types suggests that if I have an argument type of HashSet<T> I should be able to pass a frozenset to it. However, when I attempt this I get `TypeError: argument 'xyz': 'frozenset' object cannot be converted to 'PySet'`. Am I missing something — if not, I can make a small repro and open an issue.

**Comment by davidhewitt** [comment]
With apologies this is a bug that already got fixed on `main`, it'll come in 0.21: https://github.com/PyO3/pyo3/pull/3632
